web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -213,7 +216,7 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
     ) {
       localStorage.setItem('hfiCalcPreferredFireCentre', selectedFireCentre?.name)
     }
-    const { start, end } = getDateRange(isWeeklyView, dateOfInterest)
+    const { start, end } = getDateRange(true, dateOfInterest)
```

Comment:
What does this change do? Is it fixing the data loading bug when a user is in the daily view?

Reply:
Yes. Basically, if a user changes the fire centre, we want to make 1 call to retrieve all data for the prep period